Airport Ticket Booking System/BookingSystemModels/IBookingService.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Airport_Ticket_Booking_System;
+
+public interface IBookingService
+{
+    Task CreateBookingAsync(List<Passenger> passengers, Airlines airline, Flight flight, FlightClass flightClass, PaymentType paymentType,
```

Comment:
NIT: When the request has too many params, it would be better to create another model called `CreateBookingRequest`, and pass it here instead.

Reply:
Try that for the other services as well ;)